.dockerignore

```diff
@@ -0,0 +1,12 @@
+.env
```

Comment:
Are there any special instructions for setting up a .env?  Might be helpful to chuck those in the README.

I *think* my .env is borked, causing some spectacular output!

Reply:
Hmm. My `.env` is an exact copy of `env-example` and works fine.

Reply:
Seems like there's still some work to get the `.env` actually injected into the index.html of storybook, right?

Reply:
Yep, which isn't of concern right now since we don't have auth-enabled components in Storybook yet, but will need to be a future ticket.